### ISSUE TITLE
Netty requires update to 5.0.0 to resolve vulnerability

### ISSUE THREAD

**Issue by BrHowell** [docker-java/docker-java#2251]
[CVE-2023-4586](https://nvd.nist.gov/vuln/detail/CVE-2023-4586) has been published, affecting versions of netty prior to 5.0.0

**Comment by marcospereira** [comment]
But merging https://github.com/docker-java/docker-java/pull/2245 would be good, though.

**Comment by BrHowell** [comment]
Yes I ended up discovering that withdrawal, and so we're now ignoring the CVE as we're covered by 4.1.100.Final and up.

I forgot about coming back to this issue to update, sorry. I'll close it.

**Comment by yawkat** [comment]
@BrHowell please note that 4.1.100.Final does not "cover" the CVE. The relevant netty API has remained unchanged. It is simply a false positive CVE.
